_rules/descriptive-label-cc0f0a.md

```diff
@@ -1,204 +0,0 @@
----
```

Comment:
Please move the file, rather than deleting it and creating a new one. That way we don't lose the git commit history of this file. It also makes reviewing much easier. I'm going to punt on reviewing until this is done.

Reply:
Not easily possible in git, and the way to do it rewrites shared history which is even worse.
https://stackoverflow.com/questions/2314652/is-it-possible-to-move-rename-files-in-git-and-maintain-their-history

Additionally, there is here a single file which has been copied/renamed into two files because of the split of the rule. Tracking that in git seems to be even less possible.

Reply:
Also, the commit who did the change is actually detecting a rename: https://github.com/act-rules/act-rules.github.io/pull/447/commits/d96505cdac1205d30f7baf545cd4a25d17317ae4

So, I really don't know how to do it better.

…
…
…

OK, I think I figured out what happened (maybe).

- Back then, the file was called `SC2-4-6-descriptive-label.md`, using the old style of naming files (SC + name). This PR was started at that moment.
- Then, the naming convention of files changed (to name + hash). The rule file was moved as part as this commit https://github.com/act-rules/act-rules.github.io/commit/a289b7510c398fc4366f3cf9d6bfc0bbb0c5529f#diff-59490505dddc4c7a2c672f03a7dfa31e which has since been merged.
- Simultaneously, the file was renamed inside the PR as this commit: https://github.com/act-rules/act-rules.github.io/pull/447/commits/d96505cdac1205d30f7baf545cd4a25d17317ae4 
- The hashes are different in both 😢 and anyway the double move makes some ugly merge conflict 😭 

---
Now that I have an idea of what happened, I'll try to fix it but there is a risk that the fix is going to be ugly (likely, all changes made to the file as part of this PR will be squashed in a single commit, which is not that big a deal in the long run since we squash and merge anyway…)

Reply:
@WilcoFiers I tried a couple of different way to make the move working, but it looks like I failed 😞 

As you can see, the latest commit touching these file is https://github.com/act-rules/act-rules.github.io/pull/447/commits/5fcf5890a17fffbb0ff4d9da06047136fb54c865 which show this as a "file was renamed without changes" (I used `git mv`), but the "files changed" tab of the PR still show thus as deleting+creating.

I have no clue on how to improve that and I'm giving up at that point. It seems that it is too late to repare the mess that was created by the parallel rename back in April.

Reply:
Additionally, the "History" tab of these file is listing all the commit including the move ones, from which we can branch to the history of the previous file. So I do not think that history is really lost.

Reply:
The commit history definitely is lost on this. Perhaps @kasperisager can help? `git mv` will do the trick, but you may have to revert a bunch of stuff on the PR.

Reply:
I did use `git mv` to make the move (in each of my attempts). `git mv` is *not* doing that much magic: https://stackoverflow.com/questions/47401843/git-copy-file-as-opposed-to-git-mv

When I'm running the website locally on the branch, I do see the changelog for these files going back to the first draft, so it seems to be OK.